script.mjs

```diff
@@ -0,0 +1,73 @@
+const toastSandbox = document.querySelector('#toast-sandbox');
+updateStyles('#toast-sandbox', undefined, '#toast-styler');
+
+
+function updateStyles(selector, number, target) {
```

Comment:
It seems like this would work best split into 2 (or 3?) functions, instead of one function with an if statement making it do two rather-different things.

Reply:
Pulled out replaceIDs to make this cleaner